coil-base/src/main/java/coil/ImageLoader.kt

```diff
@@ -261,18 +259,6 @@ interface ImageLoader {
             this.defaults = this.defaults.copy(allowRgb565 = enable)
         }
 
-        /**
-         * Enables adding [File.lastModified] to the memory cache key when loading an image from a [File].
-         *
-         * This allows subsequent requests that load the same file to miss the memory cache if the file has been updated.
-         * To ensure a cached image is returned synchronously, [File.lastModified] is called on the main thread.
-         *
-         * Default: true
-         */
-        fun addLastModifiedToFileCacheKey(enable: Boolean) = apply {
```

Comment:
This isn't required any more since cache key computation is done on a background thread.